src/webhooks/index.ts

```diff
@@ -58,18 +58,11 @@ export default function (
             usersService,
             event.data.object as Stripe.Subscription,
             cacheService,
+            paymentService,
             fastify.log,
             config,
           );
           break;
-        case 'payment_method.attached':
```

Comment:
Why are we removing this webhook? 

According to [Stripe's documentation](https://docs.stripe.com/api/events/types) this event is triggered when a new payment method is attached to a customer (which could happen on the first payment). Therefore, if we remove this, how can we ensure that the payment method is saved the first time?

Reply:
We are removing this because we will now handle cases that come through this event: `setup_intent.succeeded` (a few lines below in the code). Since through `/setup-intent` we will be sending the `userType` (`individual` or `business`) in the metadata to update the corresponding subscription.

Reply:
But I think `setup_intent.succeeded` is not triggered at checkout. 

This is the event we use to save the user's payment method once the payment is completed if I'm not mistaken. Have you checked that the card is saved by making a payment from the checkout?
@evillalba94

Reply:
In that case, we handle the payment at `checkout.session.completed`. The payment is automatically linked to the subscription.